wizard_builder/forms.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, *args, **kwargs):
         for item in self.items:
             question_id = 'question_%s' % item.pk
             self.fields[question_id] = item.make_field()
-            self.fields[question_id].help_text = mark_safe(item.descriptive_text + self.fields[question_id].help_text)
+            self.fields[question_id].help_text = mark_safe(item.example + '<br/><br/>' + item.descriptive_text + self.fields[question_id].help_text)
```

Comment:
its excluded on some fields, so yea it does look like you tried to get rid of it. It's still present on `When did it happen?` though, for whatever content set I'm working off of right now (either Canisius or DU)